aidants_connect_web/static/css/main.css

```diff
@@ -146,7 +146,7 @@ a.no-decoration {
   background-color: #0053b3;
 }
 .background-color-gray {
```

Comment:
Mode relou++ et super-nitpick mais ça serait cool de [choisir entre `grey` et `gray`](https://www.merriam-webster.com/words-at-play/gray-vs-grey-usage-difference) 😁

Reply:
grey it is